FinSearchUnified/Tests/Bundle/SearchBundleFindologic/QueryBuilder/QueryBuilderFactoryTest.php

```diff
@@ -434,6 +465,13 @@ public function conditionProvider()
      */
     public function testSearchNavigationQuerybuilder(ConditionInterface $condition, $key, $expected)
     {
+        if ($condition instanceof ManufacturerCondition) {
+            StaticHelperTest::createTestManufacturer([
+                'id' => $key,
+                'name' => reset($expected)[0]
```

Comment:
```suggestion
                'name' => current($expected)
```

Reply:
We need to use `reset` function, i got an error with `current`.
![image](https://user-images.githubusercontent.com/103488456/192754503-706faf0c-59a5-4896-97de-b0d1e071bd8b.png)